Testscripts/Linux/check_IB_SRIOV.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+########################################################################################################
```

Comment:
Add a description for this test script.

Reply:
Fixed in bd1b2c795f72bef54052e871f01f602a4b1d9dd3